### DISCUSSION TITLE
Setting: DNS Server Domain

### DISCUSSION THREAD

**Discussion by squromiv** [TechnitiumSoftware/DnsServer#423] [Q&A]
I am newbie to network technologies and Technitium DNS Server. Rented VPS (Ubuntu OS) with static IP address and now I want to try Technitium for two purposes:

1) Blocking function (Settings/Blocking/Block List URLs) for all devices from internet (my family, coworkers etc), which send requests to IP of VPS.

2) I use [Zerotier]https://www.zerotier.com/) in order to connect my devices from different countries to "LAN" network. So I need to
2.1 Use blocking function to these "local area" devices (set Technitium as DNS Server in Zerotier controller)
2.1 Achieve these devices by names (not only by local addresses like 10.10.11.1), which are set in Technitium "Zones".

So I started Technitium Server in the VPS by Docker Compose and the first setting I don`t know how to  set - DNS Server Domain. Should it be "technitium.local" or other? I have no possibility to pay extra money for domain name.

**Comment by ShreyasZare** [comment]
The DNS Server Domain can be anything and by default its just set to the hostname. The domain name is only used when you create a primary zone as its primary name server domain name. Other than that there is no significance for it.
